package.json

```diff
@@ -16,9 +16,9 @@
     "deploy": "cross-env INSTRUMENTATION=1 RAND=1 npm run build && npm run adb:connect && ( npm run b2g:uninstall || echo ) && npm run b2g:install",
     "cypress:lint": "eslint cypress/",
     "cypress:fix-lint": "eslint cypress/ --fix",
-    "cypress:run": "cypress run --record --key $CYPRESS_RECORD_KEY",
-    "cypress:run:firefox": "cypress run --browser firefox --record --key $CYPRESS_RECORD_KEY",
-    "cypress:run:firefox:headless": "cypress run --browser firefox --headless --record --key $CYPRESS_RECORD_KEY",
```

Comment:
what's the purpose of $CYPRESS_RECORD_KEY in every run?

Reply:
It send the recording to the cypress dashboard and attaches the results to it. 

It’s good to check for test analysis for flaky tests and reduce testing time. 

Not necessary for normal testing though, it’s a nice extra